lib/spacex/launchpad.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SPACEX
+  class Launchpad < Resource
+    property 'id'
+    property 'status'
+    property 'location'
+    property 'vehicles_launched'
+    property 'details'
+    property 'site_id'
+    property 'site_name_long'
+
+    def self.all
```

Comment:
This works but is not consistent with the rest of the classes where `get` makes a determination of whether to retrieve_all or a specific entity. Also the method name is `retrieve_all` in other classes instead of all.